WPFHospitalEditor/Building.xaml.cs

```diff
@@ -19,11 +23,14 @@ namespace WPFHospitalEditor
     public partial class Building : Window
     {
         AllMapObjects allMapObjects = new AllMapObjects();
-        Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
-        HospitalMap hospitalMap = new HospitalMap();
+        public Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
+        public HospitalMap hospitalMap = new HospitalMap();
+        public int buildingId;
 
-        public Building(int id)
+
+        public Building(int id, int selectedFloor)
```

Comment:
You must use method for pulling specific data from file. You should not use if statements and not flexible methods for that task.

Reply:
It was the same problem as loading from file in AllMapObjects, and it is fixed now.